r/DESCRIPTION

```diff
@@ -4,6 +4,7 @@ Version: 0.0.0.9000
 Authors@R: c(
     person("Romain", "François", email = "romain@rstudio.com", role = c("aut", "cre")),
     person("Javier", "Luraschi", email = "javier@rstudio.com", role = c("ctb")),
+    person("Jeffrey", "Wong", email = "jeffreyw@netflix.com", role = c("ctb")),
```

Comment:
I don't know that we need to maintain these bylines

Reply:
Not necessarily.